tests/test/export/data/tests.fmf

```diff
@@ -0,0 +1,31 @@
+test: 'true'
+
+/enabled:
+    /default:
+        summary: This test is enabled by default
+    /defined:
+        summary: This test is enabled explicitly
+        enabled: true
+    /disabled:
+        summary: This test is disabled
+        enabled: false
+
+/tag:
+    /default:
+        summary: No tag by default
+    /defined:
+        summary: Custom tag defined
+        tag: slow
+    /empty:
+        summary: Explicit null tag
+        tag:
+
+/tier:
+    /default:
+        summary: No tier by default
+    /one:
+        summary: Explicit tier level
+        tier: 1
+    /two:
+        summary: Explicit tier level
+        tier: 2
```

Comment:
Why are you including 9 tests and then checking them all in the very same way? Wouldn't two or three tests be just enough? Doing basically the same checks does not bring any benefit, just makes the test slower.

Reply:
I just copied it from tests/test/select/data/tests.fmf. Now there are only two cases.

```
test: 'true'

/enabled:
    /default:
        summary: This test is enabled by default
    /disabled:
        summary: This test is disabled
        enabled: false
```

Reply:
Good, thanks!